Dataset.py

```diff
@@ -259,6 +259,18 @@ def get_seq_length(self, seq_idx):
     """
     raise NotImplementedError
 
+  def get_estimated_seq_length(self, seq_idx):
+    """
+    In contrast to self.get_seq_length(), this method is designed to work for sequences that have not been loaded yet
+    via self.load_seqs().
+    Used by meta-datasets for sequence ordering.
+
+    :param int seq_idx:
+    :rtype: NumbersDict
```

Comment:
I thought that this function was supposed to return an `int`, to be used by `get_seq_order_for_epoch`. Your function is more generic. Which is nice. But it might be problematic in some cases, e.g. when a dataset can give an estimate for only one data key (e.g. `"classes"` but not `"data"`). How would you handle that? Maybe this can return a `NumbersDict` for only the known data keys. But this should be properly documented then.